omod/src/main/webapp/resources/js/custom/Custom.js

```diff
@@ -68,105 +102,108 @@ function updateControls(status){
         $('#otherIdentifiers').hide();
         $('#registrationForm').hide();
         $('#updatePatient').hide();
-
+        $('#searchResults').hide();
     }else if(status ==1){
```

Comment:
@nribeka There is already inline documentation provided for each case. Did you mean any other type of documentation?